frontend/src/pages/RecipientRecord/pages/PrintGoals.js

```diff
@@ -0,0 +1,71 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import ReactRouterPropTypes from 'react-router-prop-types';
+import { Alert } from '@trussworks/react-uswds';
+import { getRecipientGoals } from '../../../fetchers/recipient';
+import PrintableGoal from './components/PrintableGoal';
+import PrintToPdf from '../../../components/PrintToPDF';
+import './PrintGoals.css';
+
+export default function PrintGoals({ location, recipientId, regionId }) {
+  const [goals, setGoals] = useState([]);
+  const [loading, setLoading] = useState(true);
+  const [, setError] = useState('');
+
+  useEffect(() => {
+    const sortConfig = location.state && location.state.selectedGoals
+      ? location.state.selectedGoals
+      : [];
+
+    async function fetchGoals() {
```

Comment:
This fetchGoals function would be replaced with a function that took an array of selected goal ids from the location state and passed those as request parameters to the backend